src/main/java/org/opensearch/ad/util/RestHandlerUtils.java

```diff
@@ -230,4 +236,49 @@ public static boolean isProperExceptionToReturn(Throwable e) {
     private static String coalesceToEmpty(@Nullable String s) {
         return s == null ? "" : s;
     }
+
+    public static final String unrecognized(RestRequest request, Set<String> invalids, Set<String> candidates, String detail) {
```

Comment:
This method was a part of `BaseRestHandler` to reduce the diff and maintain the consistency we should add it in `BaseExtensionRestHandler` if not present already and pull it from there.